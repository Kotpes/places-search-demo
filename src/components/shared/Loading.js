import React from 'react'
import spinner from '../../assets/Eclipse-1s-200px.svg'
import css from './Loading.module.css'

const Loading = () => {
  return (
    <div className={css.loading}>
      <img src={spinner} alt="Loading"/>
    </div>
  )
}

export default Loading