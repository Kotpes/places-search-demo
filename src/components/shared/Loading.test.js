import React from 'react'
import TestRenderer from 'react-test-renderer'
import Loading from './Loading'

it(`Matches the snapshot`, () => {
  const component = TestRenderer.create(<Loading />)  
  expect(component.toJSON()).toMatchSnapshot()
})