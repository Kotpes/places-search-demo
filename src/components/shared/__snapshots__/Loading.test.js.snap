// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<div
  className="loading"
>
  <img
    alt="Loading"
    src="Eclipse-1s-200px.svg"
  />
</div>
`;
